plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -131,6 +133,9 @@ public static GlueClient createGlueClient(GlueHiveMetastoreConfig config, OpenTe
                         .build().newExecutionInterceptor())
                 .retryStrategy(retryBuilder -> retryBuilder
                         .retryOnException(throwable -> throwable instanceof ConcurrentModificationException)
+                        .backoffStrategy(BackoffStrategy.exponentialDelay(
+                                Duration.succinctDuration(20, TimeUnit.MILLISECONDS).toJavaTime(),
```

Comment:
For this use case, `new Duration()` is sufficient. However, it would be better to do `java.time.Duration.ofMillis(...)`. We use Airlift duration when needed for configuration purposes.

Reply:
done